src/subcommand.rs

```diff
@@ -485,12 +487,37 @@ impl Subcommand {
         let mut line_width = UnicodeWidthStr::width(*name);
 
         for parameter in &recipe.parameters {
-          if let Some(evaluator) = &mut evaluator {
-            let parameter = parameter.evaluate_default(evaluator)?;
-          };
-          line_width += UnicodeWidthStr::width(
-            format!(" {}", parameter.color_display(Color::never())).as_str(),
-          );
+          // FIXME: unwrap -> ?
+          if let Some(evaluated) = evaluator
+            .as_mut()
+            .map(|evaluator| parameter.evaluate_default(evaluator))
+            .transpose()
+            .unwrap()
+            .flatten()
+          {
+            use std::collections::btree_map::Entry;
+
+            line_width += UnicodeWidthStr::width(
+              format!(" {}", evaluated.color_display(Color::never())).as_str(),
+            );
```

Comment:
We could technically skip this string alloc by simply counting the evaluated and name length + 3, but optimizations are not the point of this PR. I want to keep the diff minimal.